metrics/sinks/hawkular/client.go

```diff
@@ -138,83 +138,33 @@ func (self *hawkularSink) idName(ms *core.MetricSet, metricName string) string {
 	return strings.Join(n, separator)
 }
 
-func (self *hawkularSink) nodeName(ms *core.MetricSet) string {
-	if &self.labelNodeId != nil {
-		if v, found := ms.Labels[self.labelNodeId]; found {
+func (h *hawkularSink) nodeName(ms *core.MetricSet) string {
+	if &h.labelNodeId != nil {
```

Comment:
I've just seen this in a few other places actually, unrelated to this PR - all probably need updating.